test/dynamic_dsl.jl

```diff
@@ -455,4 +455,13 @@ end
             logpdf(normal, x, 0, 1) + logpdf(normal, y, 0, 2))
 end
 
+@testset "docstrings" begin
+    @doc "my documentation" (@gen function foo(x)
```

Comment:
Is the syntax
```julia
"""
my documentation
"""
@gen function foo(x)
  return x + 1
end
```
supported?

Reply:
Yes. I also just commited a change to the test that docuemnts it using that syntax instead.